### PR TITLE
Issue 34: Change reverse_transform method in NumberTransformer to handle nans

### PR DESCRIPTION
**Pull request by amontanez24**
Resolves #35 . This PR adds a check to the reverse_transform method of the dt.transformers.NumberTransformer class to make sure that nans are accounted for. If a nan is seen, then the default value of the transformer should be used.

### CODE REVIEW

File: tests/rdt/transformers/test_NumberTransformer.py [hunk 1]
```diff
@@ -79,6 +79,29 @@ def test_reverse_transform(self):
...
+            'subtype': 'integer',
+            'type': 'number'
+        }
+        transformer = NumberTransformer()
+        transformer.fit_transform(col, col_meta, False)
+        col2 = pd.Series([34, 23, 27, 31, np.nan], name='age')
+
+        # Run
+        expected = pd.DataFrame({'age': [34, 23, 27, 31, transformer.default_val]})
+        print(expected)
```
Comment (ManuelAlvarezC) [inline tests/rdt/transformers/test_NumberTransformer.py]: There should be any `print` or debbuging statements on the unittests.

File: tests/rdt/transformers/test_NumberTransformer.py [hunk 2]
```diff
@@ -79,6 +79,29 @@ def test_reverse_transform(self):
         # Check
         assert result.equals(expected_result)
 
+    def test_reverse_transform_nan(self):
+        """FIXME: finish when number and null transformer are compatible"""
```
Comment (ManuelAlvarezC) [inline tests/rdt/transformers/test_NumberTransformer.py]: If there's something that is pending to be done, it's better to create an issue for it than leave a comment on the code.
Also, leave always the first line for the docstring.

File: tests/rdt/transformers/test_NumberTransformer.py (hunk 1 above)
Comment (ManuelAlvarezC) [reply tests/rdt/transformers/test_NumberTransformer.py]: What I mean is that:
- If you were using the print statement for debugging, then it shouldn't have been commited.
- If you were using the print statement to check something, then it should changed to an assertion.

File: tests/rdt/transformers/test_NumberTransformer.py [hunk 3]
```diff
@@ -79,6 +79,29 @@ def test_reverse_transform(self):
...
+        }
+        transformer = NumberTransformer()
+        transformer.fit_transform(col, col_meta, False)
+        col2 = pd.Series([34, 23, 27, 31, np.nan], name='age')
+
+        # Run
+        expected = pd.DataFrame({'age': [34, 23, 27, 31, transformer.default_val]})
+        print(expected)
+        result = transformer.reverse_transform(col2, col_meta, False)
+        print(result)
```
Comment (ManuelAlvarezC) [inline tests/rdt/transformers/test_NumberTransformer.py]: Same here

File: tests/rdt/transformers/test_NumberTransformer.py (hunk 2 above)
Comment (amontanez24) [reply tests/rdt/transformers/test_NumberTransformer.py]: Fixed by https://github.com/HDI-Project/RDT/pull/37/commits/de3e452437d1b4cc76a6888e9f95dd4304c8e118

File: tests/rdt/transformers/test_NumberTransformer.py (hunk 1 above)
Comment (amontanez24) [reply tests/rdt/transformers/test_NumberTransformer.py]: Fixed by https://github.com/HDI-Project/RDT/pull/37/commits/de3e452437d1b4cc76a6888e9f95dd4304c8e118

File: tests/rdt/transformers/test_NumberTransformer.py (hunk 3 above)
Comment (amontanez24) [reply tests/rdt/transformers/test_NumberTransformer.py]: Fixed by https://github.com/HDI-Project/RDT/pull/37/commits/de3e452437d1b4cc76a6888e9f95dd4304c8e118

**Pull request merged**